package.json

```diff
@@ -94,7 +94,6 @@
     "webpack-bundle-analyzer": "^4.10.2",
     "webpack-cli": "^5.1.4",
     "webpack-dev-server": "^5.1.0",
-    "webpack-inline-manifest-plugin": "^4.0.1",
```

Comment:
It would be good to have in the commit message why this is no longer needed.

Reply:
Sounds good. I can't change the commit message without rebasing and force-pushing, which will make it harder to review. But I will make sure to do that before merging.

```
Remove webpack-inline-manifest-plugin

webpack-inline-manifest-plugin only supports webpack v4
and manifest inlining is not necessary with webpack v5 and
HtmlWebpackPlugin v5. Scripts and styles are automatically
included in the HTML file.
```